pkg/virt-launcher/virtwrap/device/hostdevice/usb/hostdev.go

```diff
@@ -0,0 +1,40 @@
+package usb
+
+import (
+	"os"
+	"strings"
+
+	v1 "kubevirt.io/api/core/v1"
+	"kubevirt.io/client-go/log"
+
+	"kubevirt.io/kubevirt/pkg/util"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+)
+
+func CreateHostDevices(vmiHostDevices []v1.HostDevice) ([]api.HostDevice, error) {
+	hostdevices := []api.HostDevice{}
+	for _, device := range vmiHostDevices {
+		env := util.ResourceNameToEnvVar("USB", device.DeviceName)
+		addressString, ok := os.LookupEnv(env)
+		if !ok {
+			// USB is only part of HostDevices. We can skip if we don't find it.
+			log.Log.V(5).Infof("USB environment variable for %s not found", device.DeviceName)
```

Comment:
We should probably increase the verbosity even more.

Reply:
From my POV, it is very hard to pinpoint the right verbosity level. Shameless plug into this https://github.com/kubevirt/kubevirt/pull/9398 so we can define, project wide, an easy convention for this.

Still, what number you consider reasonable for verbosity of debug level like this one? I don't mind changing it.